### PR TITLE
Enhancement/skale 2491 Fix metrics/bounty commands

### PR DESCRIPTION
**Pull request by alexgex**

### CODE REVIEW

File: core/metrics.py [hunk 1]
```diff
@@ -1,19 +1,31 @@
...
 from utils.filter import SkaleFilter
-import sys
 
 BLOCK_CHUNK_SIZE = 1000
 
 
-def get_nodes_for_validator(val_id):
-    skale = init_skale_from_config()
+def check_if_node_is_registered(skale, node_id):
+    if node_id in skale.nodes_data.get_active_node_ids():
```
Comment (dmytrotkk) [inline core/metrics.py]: could be simplified to `return node_id in skale.nodes_data.get_active_node_ids()`

File: core/metrics.py [hunk 2]
```diff
@@ -164,17 +180,17 @@ def get_bounty_from_events(node_ids, start_block=None, end_date=None, limit=None
...
+        bounty_row = bounty_to_ordered_row(cur_month_record, node_ids, wei)
         total_bounty += bounty_row[1]
         bounty_rows.append(bounty_row)
-    return bounty_rows, total_bounty
+    return bounty_rows
 
 
-def to_skl(digits):  # convert to SKL
-    return digits / (10 ** 18)
+def to_skl(wei):  # convert to SKL
```
Comment (dmytrotkk) [inline core/metrics.py]: use `to_skl` from `utils/helper.py`

**Review by dmytrotkk — CHANGES_REQUESTED** [review]

File: core/metrics.py (hunk 1 above)
Comment (alexgex) [reply core/metrics.py]: done

File: core/metrics.py (hunk 2 above)
Comment (alexgex) [reply core/metrics.py]: done

**Pull request merged**